src/stores/ui.ts

```diff
@@ -159,6 +160,10 @@ export const useUiStore = defineStore('ui', {
 
     updateCookies() {
       const opts = cookieOptions
+      if (useFeatureFlagStore().isOn('new_header')) {
+        opts.sameSite = 'none'
+      }
+
```

Comment:
I think this may need some cleanup. Is `opts` being used anywhere?

Reply:
Yes, it's used in the `$cookies.setAll` call below. But it wasn't setting the options correctly.